docusaurus/docs/learn/formatting-demo.md

```diff
@@ -0,0 +1,66 @@
+---
+draft: 'true'
+---
+
+# Code Block Formatting Demo
+
+This here is a "plain" code block
+```

```

Comment:
it will help to distinguish fence start and end if we model and generally require a non-empty value for lang, e.g., ` ```text`